commit d1778055df4161a038d4d8a6fb305334148aaf63 (refs/changes/48/3748/3)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2018-03-30T23:12:18+00:00 (1 year, 6 months ago)
    
    joyent/node-watershed#6 Typo in Watershed.connect() example in README.md
    Reviewed by: Alex Wilson <alex.wilson@joyent.com>
    Approved by: Alex Wilson <alex.wilson@joyent.com>

diff --git a/README.md b/README.md
index 0e96daf..057f63f 100644
--- a/README.md
+++ b/README.md
@@ -70,7 +70,7 @@ a node.js `http.Client`.  For example:
 
 ```javascript
 var shed = new Watershed();
-var wskey = ws.generateKey();
+var wskey = shed.generateKey();
 var options = {
     port: 8082,
     hostname: '127.0.0.1',
